src/utils/byte_functions/string.js

```diff
@@ -60,6 +60,220 @@ export function getStringBytes(string, addZero = false, ensureEven = false)
     return arr;
 }
 
+/**
```

Comment:
```suggestion
/**
 * This function decodes an UTF-8 string without using TextDecoder. This is done because AudioWorkletGlobalScope does not support text decoders.
```
Maybe add a comment to explain why aren't we using text decoders.